__tests__/app.test.js

```diff
@@ -195,4 +195,50 @@ describe("GET /api/reviews", () => {
             expect(response.body.reviews).toBeSortedBy("created_at", {descending: true})
     })
 })
+})
+
+describe("GET /api/reviews/:review_id/comments", () => {
+    test("GET:200 sends an array of comments by review id to the client, or an empty array if there are no comments", () => {
+        return request(app)
+        .get("/api/reviews/3/comments")
+        .expect(200)
+        .then((response) => {
+            const output = response.body.comments
+            output.forEach((comment) => {
+                expect.objectContaining({
+                    comment_id: expect.any(Number),
+                    votes: expect.any(Number), created_at: expect.toBeDate(), author: expect.any(String),
+                    body: expect.any(String),
+                    review_id: expect.any(Number)})
+                })
+                expect(output.length).toBeGreaterThan(1)
```

Comment:
You should also be precise about how many comments you expect back (line 214) - you have access to the test data you know how many should be in this array